PocketKit/Sources/PocketKit/Article/Presenters/ArticleComponentPresenter.swift

```diff
@@ -9,4 +9,5 @@ protocol ArticleComponentPresenter {
     func cell(for indexPath: IndexPath, in collectionView: UICollectionView) -> UICollectionViewCell
     func size(for availableWidth: CGFloat) -> CGSize
     func clearCache()
+    var highlights: [ArticleComponentHighlight] { get }
```

Comment:
This will likely change to avoid defining highlights for components that don't have them